libs/entitlement/src/main/java/org/elasticsearch/entitlement/initialization/EntitlementInitialization.java

```diff
@@ -92,7 +92,12 @@ private static PolicyManager createPolicyManager() throws IOException {
             "server",
             List.of(new Scope("org.elasticsearch.server", List.of(new ExitVMEntitlement(), new CreateClassLoaderEntitlement())))
         );
-        return new PolicyManager(serverPolicy, pluginPolicies, EntitlementBootstrap.bootstrapArgs().pluginResolver());
+        return new PolicyManager(
+            serverPolicy,
+            pluginPolicies,
+            EntitlementBootstrap.bootstrapArgs().pluginResolver(),
+            PolicyManager.class.getModule()
```

Comment:
could you add a comment that this is just the entitlement module? it looks confusing otherwise